### PR TITLE
fix: inpsect image size from layer data

### PR DESCRIPTION
**Pull request by kemingy**
Signed-off-by: Keming <kemingyang@tensorchord.ai>

### CODE REVIEW

File: pkg/server/environment_create.go [hunk 1]
```diff
@@ -254,6 +256,13 @@ func getImageMeta(ctx context.Context, imageName string) (
 	if err != nil {
 		return
 	}
+	if size < 0 {
```
Comment (gaocegege) [inline pkg/server/environment_create.go:259]: I think the total size should be in the image metadata. Let's me check it.

Comment (kemingy) [reply pkg/server/environment_create.go:259]: `image.Size()` always returns -1. Not sure why.

**Comment by gaocegege** [comment]
Let's merge it first. Please open an issue to keep track of the size thing. Thanks!

**Pull request merged**